prow/config/config.go

```diff
@@ -135,6 +136,31 @@ type ProwConfig struct {
 	DefaultJobTimeout *metav1.Duration `json:"default_job_timeout,omitempty"`
 }
 
+// PresubmitsStatic returns the presubmits in Prows main config.
+// **Warning:** This does not return dynamic Presubmits configured
+// inside the code repo, hence giving an incomplete view. Use
+// `GetPresubmits` instead if possible.
+func (c *Config) PresubmitsStatic() map[string][]Presubmit {
```

Comment:
nit: can we do `StaticPresubmits`? would be more idiomatic in English

Reply:
Yeah it sounds better, the reason to name it like that is that if ppl that dont know the codebase that well want to get presubmits and check for funcs starting with P on a `*Config` in their IDE they wouldn't get `StaticPresubmits` as suggestion